ColorDeductoActivity.py

```diff
@@ -84,7 +73,12 @@ def __init__(self, handle):
 
         self._sharing = False
         self._initiating = False
-        self._setup_presence_service()
+        self.connect('shared', self._shared_cb)
+        self.connect('joined', self._joined_cb)
+        self._collab = CollabWrapper(self)
```

Comment:
Since this was also used in another file, you can make it an instance variable rather it be a private one.

Reply:
Yeah I will try this.